css_tasks/task10/scss/_general.scss

```diff
@@ -0,0 +1,268 @@
+@import "variables";
+@import "mixins";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: $primaryFont;
+  color: $mainTextColor;
+}
+
+.container {
+  padding: 0 10%;
+}
+picture {
+  display: block;
+}
+.header {
+  &__headline {
+    @include headline;
+  }
+  &__headline--center {
+    text-align: center;
+  }
+  &__title {
+    @include headerText;
+    .header__dots {
+      color: $mainThemeColor;
+    }
+  }
+  &__title--center {
+    text-align: center;
+  }
+  &__title--form {
+    padding-bottom: 6rem;
+    color: #fff;
+  }
+  &__text {
+    padding-bottom: 1.5rem;
+    font-size: $mainTextFontSize;
+    @include textDisplay;
+  }
+  &__text--features-secondary {
+    width: 50%;
+    margin: 0 auto;
+    padding-bottom: 4.8125rem;
+    text-align: center;
+  }
+}
+.inner-container {
+  display: flex;
+  &__element {
+    width: 30%;
+    margin: 0 auto;
+    h3 {
+      padding-top: 1.5rem;
+      @include elementTitle;
+    }
+    p {
+      padding-bottom: 11.5rem;
+      @include elementText;
+    }
+  }
+  &__element--testimonials {
+    div {
+      margin: 6.875rem auto 0;
```

Comment:
would better to create some padding-top for container then add margin for every element inside each block